first-module/src/main/java/firstmodule/controller/CommandController.java

```diff
@@ -0,0 +1,55 @@
+package firstmodule.controller;
+
+import firstmodule.domain.CommandDomain;
+import firstmodule.repository.CommandRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Random;
+import java.util.UUID;
+
+@RestController
+@RequestMapping(value = "/command")
+public class CommandController {
+
+    @Autowired
+    private CommandRepository commandRepository;
+
+    @Autowired
+    private KafkaTemplate<String, CommandDomain> kafkaTemplate;
+
+    @GetMapping(value = "/add")
```

Comment:
Creation of an entity should be done using Post

Reply:
@ktereshchenko I used Get only for testing through browser

Reply:
Got it. It is sometimes easier indeed, but not always possible. Try using `curl` from command line or Postman - the standalone app for these purposes. However the best would be to write the test on it using wiremock (http://wiremock.org/docs/stubbing/)